test/java/com/rapleaf/jack/TestModelWithID.java

```diff
@@ -33,6 +34,9 @@ public void setUp() {
 
   public void testFields() {
 
+    assertTrue(postModel.toString().equals("<Post id: 0 title: Test Post posted_at_millis: 100 user_id: 1 updated_at: 0>"));
```

Comment:
for regression testing purposes, can you have a test that verifies the ID is something real after it's saved?  you can re-fetch the model after save(userModel); is called and it should have a real ID